coolwsd.xml.in

```diff
@@ -258,6 +258,8 @@
         <url desc="URL of optional JSON file that lists fonts to be included in Online" type="string" default=""></url>
     </remote_font_config>
 
+    <user_stats desc="Enable user stats. i.e: logs the details of a file and user" type="bool" default="false">false</user_stats>
```

Comment:
We already have a setting to print stats in the patch - I pointed you to - we should include this config key in the same block - can you move it there ?

Thanks !